.github/workflows/cypress.yaml

```
@@ -0,0 +1,148 @@
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  Cleanup:
+    if: always() && github.repository == 'palisadoesFoundation/talawa-admin'
+    needs: Setup-Backend-Run-Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete build artifact
+        uses: geekyeggo/delete-artifact@v5
+        with:
+          name: build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)